LayoutTests/imported/w3c/web-platform-tests/trusted-types/should-sink-type-mismatch-violation-be-blocked-by-csp-001-expected.txt

```diff
@@ -0,0 +1,68 @@
+CONSOLE MESSAGE: This requires a TrustedHTML value else it violates the following Content Security Policy directive: "require-trusted-types-for 'script'"
+CONSOLE MESSAGE: Refused to connect to ws://common/blank.html because it does not appear in the connect-src directive of the Content Security Policy.
+CONSOLE MESSAGE: This requires a TrustedHTML value else it violates the following Content Security Policy directive: "require-trusted-types-for 'script'"
+CONSOLE MESSAGE: Refused to connect to ws://common/blank.html because it does not appear in the connect-src directive of the Content Security Policy.
+CONSOLE MESSAGE: Invalid sink group in 'require-trusted-types-for' Content Security Policy directive: ''script''script''.
+
+CONSOLE MESSAGE: Refused to connect to ws://common/blank.html because it does not appear in the connect-src directive of the Content Security Policy.
+CONSOLE MESSAGE: Invalid sink group in 'require-trusted-types-for' Content Security Policy directive: ''invalid''.
+
+CONSOLE MESSAGE: This requires a TrustedHTML value else it violates the following Content Security Policy directive: "require-trusted-types-for 'script'"
+CONSOLE MESSAGE: Refused to connect to ws://common/blank.html because it does not appear in the connect-src directive of the Content Security Policy.
+CONSOLE MESSAGE: Invalid sink group in 'require-trusted-types-for' Content Security Policy directive: ''invalid''.
+
+CONSOLE MESSAGE: This requires a TrustedHTML value else it violates the following Content Security Policy directive: "require-trusted-types-for 'script'"
+CONSOLE MESSAGE: Refused to connect to ws://common/blank.html because it does not appear in the connect-src directive of the Content Security Policy.
+CONSOLE MESSAGE: Invalid sink group in 'require-trusted-types-for' Content Security Policy directive: ''invalid''.
+
+CONSOLE MESSAGE: Invalid sink group in 'require-trusted-types-for' Content Security Policy directive: ''also-invalid''.
+
+CONSOLE MESSAGE: This requires a TrustedHTML value else it violates the following Content Security Policy directive: "require-trusted-types-for 'script'"
+CONSOLE MESSAGE: Refused to connect to ws://common/blank.html because it does not appear in the connect-src directive of the Content Security Policy.
+
+FAIL Multiple enforce require-trusted-types-for directives. assert_equals: expected 5 but got 1
+PASS Multiple report-only require-trusted-types-for directives.
+FAIL One violated report-only require-trusted-types-for directive followed by multiple enforce directives assert_equals: expected 5 but got 1
+FAIL One violated enforce require-trusted-types-for directive followed by multiple report-only directives assert_equals: expected 5 but got 1
+FAIL Mixing enforce and report-only require-trusted-types-for directives. assert_equals: expected 6 but got 1
```

Comment:
Do we have some kind of follow-up filed for these failures?

Reply:
Not yet, I can do though. I'm not entirely sure if they are test issues on WebKit ones.

Reply:
Typically, I've just found what I think is the cause, I'll make a follow up patch to fix that. Per the spec the test is correct.